client-src/elements/form-field-enums.js

```diff
@@ -133,11 +133,18 @@ export const DEPRECATED_FIELDS = new Set([
   'standardization',
 ]);
 
+export const GATE_TYPES = {
```

Comment:
Are these searchable fields for the chromedash-feature-filter? if so, should we add corresponding fields to the backend as we..?

Reply:
No, I don't think we need to search on gate type.